src/game/Maps/Map.cpp

```diff
@@ -874,7 +874,12 @@ void Map::Update(const uint32& t_diff)
 #endif
 
     // Send world objects and item update field changes
-    SendObjectUpdates();
+    m_clientUpdateTimer += t_diff;
```

Comment:
this doesn't seem to break anything and *roughly* aligns with packet data I've seen.
I don't know if this is because ABK servers were just way slower with processing ticks or if there is actually some limit like this

Reply:
Object update tick is 400. Didnt manage to slot that in into our system because its potentially dangerous.

Reply:
should I remove it from the PR or should I correct the value?